components/Dialogs/src/private/MDCAlertControllerView+Private.h

```diff
@@ -56,69 +56,67 @@
 @property(nonatomic, assign) BOOL adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable;
 
 /**
- The margins around the title icon or the title icon view against the dialog
- edges (top, leading, trailing) and the title (bottom). Note that the actual
- bottom space is the smallest between titleImageInsets.bottom and
- titleInsets.top.
+ Weather adjustable insets mode is enabled for the dialog view. If set to @c
+ YES, a new layout calculation that is customizable by the insets in this header
+ file is used to layout the dialog. If set to @c NO, we fall back to the legaccy
+ layout calculation, ignoring all customized inset values.
+
+ Default value is @c NO.
+ */
+@property(nonatomic, assign) BOOL enableAdjustableInsets;
+
+/**
+The edge insets around the title icon or title icon view against the dialog
+ edges (top, leading, trailing) and the title (bottom). Note that
+ `titleIconInsets.bottom` overrides `titleInsets.top`.
```

Comment:
Given that "overrides" means something very specific in OO programming, I think I would change this to "`titleIconInsets.bottom` takes precedence over `titleInsets.top`"

Reply:
Done